AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerpolicy/DefaultActiveSpeakerPolicy.swift

```diff
@@ -9,19 +9,28 @@
 import Foundation
 
 @objcMembers public class DefaultActiveSpeakerPolicy: NSObject, ActiveSpeakerPolicy {
+    public static let defaultSpeakerWeight = 0.9
```

Comment:
Few questions, in swift, can't they specify which to input? For instance, they can set weight but sets others as default right? Wouldn't this break that case?

Reply:
They are still use the other init with all three params